packages/grid/src/vaadin-grid-dynamic-columns-mixin.js

```diff
@@ -90,6 +90,13 @@ export const DynamicColumnsMixin = (superClass) =>
       return columnTree;
     }
 
+    /** @protected */
+    _debounceUpdateColumnTree() {
+      this.__updateColumnTreeDebouncer = Debouncer.debounce(this.__updateColumnTreeDebouncer, microTask, () =>
```

Comment:
unrelated note: We could consider rewriting microTask to use queueMicrotask instead of the current Polymer workaround based on MutationObserver. I believe using MutationObserver isn't quite efficient as it causes unnecessary changes in the DOM which may subsequently cause the browser repaint process and so on.

Reply:
Great find. I was under the impression that it used `MutationObserver` only as a fallback for browsers that don't support `queueMicrotask` but apparently, that's not the case.